<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutUK</class>
 <widget class="QWidget" name="LayoutUK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="KeyButton" name="qButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>й</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Й</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="wButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ц</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="eButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>у</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">У</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="rButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>к</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">К</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="tButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>е</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Е</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="yButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>н</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Н</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>г</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Г</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="splitWidget1" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="uButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ш</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ш</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="iButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>щ</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Щ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="oButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>з</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">З</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>х</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Х</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ї</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ї</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="backspace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="go-previous">
         <normaloff>../</normaloff>../</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="middleLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="KeyButton" name="aButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ф</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ф</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="sButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>і</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">І</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="dButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>в</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">В</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="fButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>а</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">А</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="gButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>п</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">П</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>р</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Р</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>о</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">О</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="splitWidget2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="hButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>л</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Л</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="jButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>д</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Д</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="kButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ж</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ж</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="lButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>є</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Є</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>'</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ReturnKey" name="returnButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>⏎</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="KeyButton" name="shift">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="input-caps-on">
         <normaloff>../</normaloff>../</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="zButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>я</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Я</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="xButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ч</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ч</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="cButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>с</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">С</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="vButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>м</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">М</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>и</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">И</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>т</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Т</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="splitWidget3" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="bButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ь</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="nButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>б</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Б</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="mButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ю</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ю</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ґ</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="shiftedOutput" stdset="0">
        <string notr="true">Ґ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="commaButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>,</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KeyButton" name="periodButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyButton</class>
   <extends>QPushButton</extends>
   <header>keybutton.h</header>
   <slots>
    <signal>held(QPoint)</signal>
    <signal>touchMoved(QPoint)</signal>
    <signal>letGo(QPoint)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ReturnKey</class>
   <extends>QPushButton</extends>
   <header>specialKeys/returnkey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
